src/pages/KubernetesError.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div>
+    <div class="page-body">
+      <h2 data-test="k8s-error-header">
+        <img id="logo" src="../../resources/icons/logo-square-red@2x.png" />
+        Kubernetes Error
+      </h2>
```

Comment:
I'd suggest that we make use of flexbox to display the image next to the h2 to make the act of laying out items a little more simple

```suggestion
      <div class="error-header">
        <img id="logo" src="../../resources/icons/logo-square-red@2x.png" />
        <h2 data-test="k8s-error-header">
          Kubernetes Error
        </h2>
      </div>
```

and in our `style` (reduced the space between the icon and h2 to `0.75` or `1`)

```
  .error-header {
    display: flex;
    gap: 0.75rem;
  }
```

Reply:
We also might need to adjust margins and line height of h2 to get the image and text to vertically align properly

Reply:
Done, thanks.